pkgs/games/nottetris2/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub, zip, love_0_7, makeWrapper, makeDesktopItem }:
+
+let
+  pname = "nottetris2";
+  version = "2.0";
+
+  desktopItem = makeDesktopItem {
+    name = "nottetris2";
+    exec = pname;
+    comment = "It's like tetris, but it's not";
+    desktopName = "nottetris2";
+    genericName = "nottetris2";
+    categories = "Game";
+  };
+
+in
+
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "Stabyourself";
+    repo = pname;
```

Comment:
Would it be possible to `fetchurl` from https://stabyourself.net/nottetris2/ instead? The website appears to be by far more reachable on the internet than the github when searching for `nottetris2`, and I can't find a link from the website to the github — the github being identity theft, while unlikely, is unfortunately always a possibility. This being said, I notice that `orthorobot` already uses the github, so… it's probably not that bad to add one dependency on this github here, but I wonder where the “authority” was taken from when merging `orthorobot` :)

Reply:
The download on the website isn't versioned and doesn't contain a license, so I'd be in favor of sticking with GitHub.

We can also do the direct approach: @Stabyourself you're one of the people responsible for stabyourself.net, right? Can you maybe include a link there to your GitHub or clarify the situation?

Reply:
Hi, yeah I'm the dev of the games. Back then I didn't really know/care much about open source standards, so everything's a bit home-baked, sorry about that.
I changed all the "Source" links on the website to go to github instead.

Reply:
Great, thank you! :100: 

Merging, as a consequence :)